apps/app/src/views/routes/SettingsV2/apollo.tsx

```diff
@@ -0,0 +1,79 @@
+import {
```

Comment:
We have something similar in the SDK, would love to improve that tbh so we have one place we can point to